packages/office-ui-fabric-react/src/components/Rating/Rating.base.tsx

```diff
@@ -12,20 +12,24 @@ interface IRatingStarProps extends React.AllHTMLAttributes<HTMLElement> {
   disabled: boolean;
   readOnly: boolean;
   classNames: IProcessedStyleSet<IRatingStyles>;
+  icon?: string;
```

Comment:
should we split this into 2 icons? unfilled and filled?

E.g. want it to be an outlined star unfilled and a solid star filled.

Reply:
What would the desired behavior be for partially filled stars (e.g. rating of 2.5)?

Reply:
It would still do what it does today. E.g. overlay half the star. But then, not every scenario uses half stars.

But it would open up more scenarios to support.

Reply:
You could stick with `icon` and have something like `unselectedIcon` which defaults to the `icon` value if not provided. It keeps the api simple like you have.

Reply:
It's just a suggestion though. I'm good with what you have here.

Reply:
Added `unselectedIcon` property with the following behavior:

1. If neither the `icon` nor the `unselectedIcon` property is provided, all stars use the "FavoriteStarFill" icon (consistent with the behavior before this change).
<img width="182" alt="default-icons" src="https://user-images.githubusercontent.com/6687333/49677922-245c3200-fa36-11e8-8d23-fbcdf599b49e.png">

2. If only the `icon` property is provided, all stars use the `icon` property.
<img width="116" alt="icon" src="https://user-images.githubusercontent.com/6687333/49677891-f7a81a80-fa35-11e8-9cd6-cf0ede7a5013.png">

3. If only the `unselectedIcon` property is provided, filled or partially filled stars use "FavoriteStarFill" icon and unfilled stars use the `unselectedIcon` property.
<img width="109" alt="unselectedicon" src="https://user-images.githubusercontent.com/6687333/49677899-07bffa00-fa36-11e8-9de0-fdd6a341463a.png">

4. If both the `icon` and the `unselectedIcon` properties are provided, filled or partially filled stars use the `icon` property and unfilled stars use the `unselectedIcon` property.
<img width="110" alt="icon-and-unselectedicon" src="https://user-images.githubusercontent.com/6687333/49677904-0ee70800-fa36-11e8-868b-1654c9e23fa6.png">